spec/javascripts/modules/organisation-list-filter_spec.js

```diff
@@ -59,28 +59,30 @@ describe('organisation-list-filter.js', function () {
     '</div>'
 
   beforeAll(function () {
-    $(document.body).append(stubStyling)
-    $(document.body).append(form)
-    $(document.body).append(organisations)
-    GOVUK.filter.init()
+    var wrapper = $('<div>').addClass('wrapper')
```

Comment:
Hissssssssssssss there's still jQuery here! This isn't as big a problem since Jasmine loads it's own JQuery installation and it doesn't reach users so this isn't a blocker for approval. Tidying up here would make a it a little bit nicer though :)

Reply:
There's a lot of jQuery in the tests generally. We've decided to leave it for now as it doesn't cause a problem, generally helps when writing tests, and would likely more than double the effort required to remove jQuery.

Having said that, there are lots of gotchas involved in having JS code and jQuery tests, but that's a separate conversation.